src/electron/platform/android/rule-information.ts

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import { RuleResultsData } from './scan-results';
+
+export type HowToFixDelegate = (ruleResultsData: RuleResultsData) => string;
+
+export class RuleInformation {
+    constructor(readonly RuleId: string, readonly RuleDescription: string, readonly howToFixDelegate: HowToFixDelegate) {}
```

Comment:
the variables `RuleId` and `RuleDescription` should be camelCase to match other files; we may also consider `private readonly…` instead of `readonly` for these variables

Reply:
Fixed as discussed